Formula/step.rb

```diff
@@ -0,0 +1,27 @@
+class Step < Formula
+  desc "Crypto and x509 Swiss-Army-Knife"
+  homepage "https://github.com/smallstep/cli"
+  url "https://github.com/smallstep/cli/releases/download/v0.8.2/brew_step_0.8.2.tar.gz"
+  sha256 "bb5cd17fccf78a0432736fe8e32dc5805507784884a06720b3ee99eca39bf9ba"
+
+  depends_on "dep" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/smallstep").install buildpath.children
+    cd "src/github.com/smallstep/cli" do
+      system "make", "build"
+      bin.install "bin/step" => "step"
+      bash_completion.install "autocomplete/bash_autocomplete" => "step"
+    end
+    cd "src/github.com/smallstep/certificates" do
+      system "make", "build"
+      bin.install "bin/step-ca" => "step-ca"
+    end
+  end
+
+  test do
+    system "#{bin}/step", "crypto", "keypair", "foo.pub", "foo.priv", "--no-password", "--insecure"
```

Comment:
If you're installing both the Step Cli and the `step-ca` tool, would it be easy/straightforward to test `#{bin}/step-ca` as well?

Also, can you please check for the creation of the key pair files from this test, not just the return status?

Reply:
Added a couple of more tests with a few checking for the files that should have been created.

The `step-ca` command kicks off a server which the `cli` (or any other client) can then access via an API. Is kicking off a server from the test-suite acceptable? If so, then we can try delivering a dummy configuration file with the homebrew package, kicking up the server, and making a health request (or something more involved).

Reply:
Yes it is acceptable and there are several examples in the Formulae on how to do it. If you grep for `fork` you should be able to find plenty of examples.

Reply:
Thanks for the tip! Added a few tests with a forked server.